### PR TITLE
(feat) Next Gen pipelines

### PR DESCRIPTION
**Pull request by warrenbailey**
Creating a jenkins-x.yml file to enable this project to be built by the next gen pipelines.

### CODE REVIEW

File: update-bot.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+updatebot push-version --kind helm \
+    jenkinsxio/builder-aws-cdk ${VERSION} \
```
Comment (polothy) [inline update-bot.sh]: Please also add these new builders:
```
    jenkinsxio/builder-awscli ${VERSION} \
    jenkinsxio/builder-awscli-local ${VERSION} \
```

**Comment by rawlingsj** [comment]
ok builds down to 31 mins.  There's still improvements to be made but this seems like a great base now.  Downstream charts will need to reference the `gcr.io` repo but I'll sort all that out after release.

Updatebot files also need to be updated, I'll also sort this or raise issues describing what's outstanding.

**Comment by rawlingsj** [comment]
@jstrachan @garethjevans @warrenbailey @cagiti here we go!  I'll fix up downstream charts to use `gcr.io` also the platform `PodTemplates`, there maybe things I'll miss so if you hear or see anything shout and we can fix fast.

**Pull request merged**